ibis/backends/clickhouse/tests/snapshots/test_select/test_join_self_reference/out.sql

```diff
@@ -1,5 +1,17 @@
 SELECT
-  t0.*
-FROM functional_alltypes AS t0
-INNER JOIN functional_alltypes AS t1
-  ON t0.id = t1.id
\ No newline at end of file
+  t1.id,
```

Comment:
Do we want to avoid explicitly listing the columns here? If yes, we can add a utility for that in a follow-up.

Reply:
We definitely need the table name prefix here, but if there were a way we could generate `t1.*` that might be nice.

Reply:
Created a follow-up: https://github.com/ibis-project/ibis/issues/7852